MetaMask/metamask-extension
# fix: add missing update available and arrow icon

## **Description**

Adds a missing `Update Available` and arrow icon in the Add Snap Account Page

## **Manual testing steps**

1. Install outdated snap (for example, https://metamask.github.io/snap-simple-keyring/0.1.4/)
2. From wallet view select accounts drop down
3. Select Add snap account
4. At snaps grid the installed snap select CTA with copy "Configure"
5. See the update CTA

## **Screenshots/Recordings**

### **Before**
![updatesnap3](https://github.com/MetaMask/metamask-extension/assets/96463427/c71f694a-60d0-4bb9-ae67-1bb29977383b)


### **After**
<img width="1297" alt="Screenshot 2023-09-28 at 22 20 54" src="https://github.com/MetaMask/metamask-extension/assets/96463427/f1584cf8-573e-488d-86b6-ca83f8a10433">

![updatesnap2](https://github.com/MetaMask/metamask-extension/assets/96463427/d7890f4f-40ce-42e5-9736-d4a09893fd6e)
![update-snap1](https://github.com/MetaMask/metamask-extension/assets/96463427/87658f48-b1bf-464a-8bdb-db567d2291dc)


## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/46

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
